src/models/predict_lr_model.py

```diff
@@ -0,0 +1,74 @@
+# model neu laden und funktion, wo ein wert eingefügt wird
+# parameter model und ein wert
+# load_model()
+# predict_value()
+
+
+import joblib
+import pandas as pd
+import numpy as np
+
+
+def load_model() -> object:
```

Comment:
In dem Fall wird nicht ein object sondern ein imblearn.pipeline.Pipeline Wert zurückgegeben